build.gradle

```diff
@@ -1,17 +1,10 @@
-buildscript {
-  repositories {
-    jcenter()
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
-  }
-  dependencies {
-    classpath 'gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.14.0'
-    classpath 'com.github.jengelman.gradle.plugins:shadow:4.0.4'
-  }
-}
+// Copyright 2019-2020 LinkedIn Corporation. All rights reserved.
+// Licensed under the BSD-2 Clause license.
+// See LICENSE in the project root for license information.
 
 plugins {
+  id "com.diffplug.spotless" version "5.8.2"
+  id 'com.github.johnrengelman.shadow' version '2.0.4'
```

Comment:
This was previously 4.0.4, any reason to change it to 2.0.4? Also https://github.com/johnrengelman/shadow states that Gradle 6.x requires atleast version 5.2.0 of shadow. Should we upgrade?

Reply:
Upgraded to 5.2.0 in latest version of the patch.